CHANGELOG.md

```diff
@@ -22,6 +22,14 @@ Adding a new version? You'll need three changes:
 
 ## Unreleased
 
+### Breaking Changes
+
+- `KonnectExtension` `v1alpha2` has been introduced as the API does not allow anymore to
+  reference Konnect Gateway ControlPlanes via plain KonnectID.
+  Use Mirror `KonnectGatewayControlPlane`s instead.
+  [#449](https://github.com/Kong/kubernetes-configuration/pull/449)
+  [#452](https://github.com/Kong/kubernetes-configuration/pull/452)
+
```

Comment:
I guess this is unrelated to this PR, isn't it?

Reply:
It is unrelated but I felt compelled to add it. Do you feel we need a separate PR for that?

Reply:
nah, that's fine :)